app/javascript/components/enrollment/Enrollment.js

```diff
@@ -184,17 +187,28 @@ class Enrollment extends React.Component {
         this.setState({ index: lastIndex, lastIndex: null });
       });
     } else {
+      let step = 1;
+      if ((this.state.enrollmentState.isolation ? 4 : 5) == index) {
+        step = 2;
+      }
       this.setState({ direction: 'next' }, () => {
-        this.setState({ index: index + 1, lastIndex: null });
+        this.setState({ index: index + step, lastIndex: null });
```

Comment:
is there a reason this is in the callback of the other set state? it doesn't seem like it needs to be

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/ed5b1fa40df633f1cbe6d4556e87ba0f7ae8f545

Reply:
I'm still seeing this nested